python/mxnet/optimizer/optimizer.py

```diff
@@ -380,6 +381,11 @@ def set_wd_mult(self, args_wd_mult):
                     self.wd_mult[name] = float(attr[name]['__wd_mult__'])
         self.wd_mult.update(args_wd_mult)
 
+    def set_current_context(self, ctx):
+        if ctx not in self._all_index_update_counts:
```

Comment:
rename to _set_current_context, and document the type of ctx?

Reply:
Will do

Reply:
Done.